daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2126,9 +2157,11 @@ func (devices *DeviceSet) removeDevice(devname string) error {
 	return err
 }
 
-func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) error {
+func (devices *DeviceSet) cancelDeferredRemoval(info *devInfo) (bool, error) {
```

Comment:
Shouldn't these be error cases?
We are saying "go cancel this thing" and then essentially returning back "ok nothing is wrong".
We can always check the error type if need be.

I think it may be a little more clear handling this logic in the caller... this is similar to how we tend to ignore `IsNotExist` type errors when we are removing the file anyway.